.drone.yml

```diff
@@ -37,16 +37,21 @@ pipeline:
       - make test-php-lint
       - make test-php-style
 
-  phpunit:
+  setup-antivirus-environment:
     image: owncloudci/php:${PHP_VERSION}
     pull: true
     commands:
       # dummy clamav required for unit tests
       - php /var/www/owncloud/apps/files_antivirus/tests/util/avirserver.php&
       # wait shortly for the server to be available
       - wait-for-it 127.0.0.1:5555
-      # required to pass the executable tests
+      # required to pass the executable tests and acceptance tests
```

Comment:
this needs to be done in the container where the tests run to properly work. 
Otherwise you do in 
- container A - a symlink at `/usr/bin/clamscan` 
- containB doesn't know about `/usr/bin/clamscan` 

only drone workspace is shared between containers

Reply:
additional - changing the container for the remote server, also means the unit tests need to be injected with the host name of the container where the dummy clamav server is running

Reply:
Thanks for the clues!